src/features/pages/deriv-ctrader/how-to-start/data.tsx

```diff
@@ -0,0 +1,247 @@
+        <StaticImage
+            src="../../../../images/common/trading-platforms/ctrader/mobile-step-2.png"
+            alt="step"
+            loading="eager"
+            formats={['avif', 'webp', 'auto']}
+            quality={50}
+            objectFit="contain"
+            placeholder="none"
+        />
+    ),
+    step3: (
+        <StaticImage
+            src="../../../../images/common/trading-platforms/ctrader/mobile-step-3.png"
+            alt="step"
+            loading="eager"
+            formats={['avif', 'webp', 'auto']}
+            quality={50}
+            objectFit="contain"
+            placeholder="none"
+        />
+    ),
+    'step2-demo': (
+        <StaticImage
+            src="../../../../images/common/trading-platforms/ctrader/mobile-step-2-demo.png"
+            alt="step"
+            loading="eager"
+            formats={['avif', 'webp', 'auto']}
+            quality={50}
+            objectFit="contain"
+            placeholder="none"
+        />
+    ),
+    'step3-demo': (
+        <StaticImage
+            src="../../../../images/common/trading-platforms/ctrader/mobile-step-3-demo.png"
+            alt="step"
+            loading="eager"
+            formats={['avif', 'webp', 'auto']}
+            quality={50}
+            objectFit="contain"
+            placeholder="none"
+        />
+    ),
+}
+export const demo: TradingPlatformStepperViewType = [
+    {
+        title: '_t_<0>Sign in</0> to your Deriv account. If you don’t have one, <1>sign up</1> for free._t_',
+        image: how_to_start_images['step1'],
+        component: [
+            <Link
+                textcolor="brand"
+                key={0}
+                url={{
+                    type: 'internal',
+                    to: '/',
+                }}
+            />,
+            <Link
+                textcolor="brand"
+                key={1}
+                url={{
+                    type: 'internal',
+                    to: '/',
+                }}
+            />,
+        ],
+    },
+    {
+        title: '_t_Go to the Trader’s hub and choose the Demo option._t_',
+        image: how_to_start_images['step2-demo'],
+    },
+    {
+        title: '_t_Under CFDs, look for Deriv cTrader and select Get._t_',
+        image: how_to_start_images['step3-demo'],
+    },
+]
+export const demo_mobile: TradingPlatformStepperViewType = [
+    {
+        title: '_t_<0>Sign in</0> to your Deriv account. If you don’t have one, <1>sign up</1> for free._t_',
+        image: how_to_start_images_mobile['step1'],
+        component: [
+            <Link
+                textcolor="brand"
+                key={0}
+                url={{
+                    type: 'internal',
+                    to: '/',
+                }}
+            />,
+            <Link
+                textcolor="brand"
+                key={1}
+                url={{
+                    type: 'internal',
+                    to: '/',
+                }}
+            />,
```

Comment:
You have to update links here
Please refer this changes if you don't know how 
https://github.com/binary-com/deriv-com/pull/5055/files#diff-1d84979744b3173d0c46889949d33910051b5b8e1d43f4cc4d8cadebf28bfa90
https://github.com/binary-com/deriv-com/pull/5055/files#diff-118a1ed2ebf5b946c94c5dfeae1c4d13c9a57ef105b36ad323e0b7b8e368da27
https://github.com/binary-com/deriv-com/pull/5055/files#diff-817e32aa05e78052c2733ded7a02e77a8c142a0be0975a16e32e697a64661cd6